app/views/learning_hours_mailer/learning_hours_report_email.html.erb

```diff
@@ -0,0 +1,80 @@
+  <style>/* Email styles need to be inline, except @media queries */</style>
+  <style type="text/css">
+    @media only screen and (max-width: 640px) {
+      body {
+        padding: 0 !important;
+      }
+
+      h3 {
+        font-weight: 800 !important;
+        margin: 20px 0 5px !important;
+      }
+
+      h3 {
+        font-size: 16px !important;
+      }
+
+      .container {
+        padding: 0 !important;
+        width: 100% !important;
+      }
+    }
+  </style>
+</head>
+
+<body itemscope itemtype="http://schema.org/EmailMessage" style="font-family: 'Helvetica Neue',Helvetica,Arial,sans-serif; box-sizing: border-box; font-size: 14px; -webkit-font-smoothing: antialiased; -webkit-text-size-adjust: none; width: 100% !important; height: 100%; line-height: 1.6em; background-color: #f6f6f6; margin: 0;" bgcolor="#f6f6f6">
+  <table class="main-body" style="box-sizing: border-box; min-height: 150px; padding-top: 5px; padding-right: 5px; padding-bottom: 5px; padding-left: 5px; width: 100%; height: 100%; background-color: rgb(234, 236, 237);" width="100%" height="100%" bgcolor="rgb(234, 236, 237)">
+    <tbody style="box-sizing: border-box;">
+      <tr class="row" style="box-sizing: border-box; vertical-align: top;" valign="top">
+        <td class="main-body-cell" style="box-sizing: border-box;">
+          <table class="container" style="box-sizing: border-box; font-family: Helvetica, serif; min-height: 150px; padding-top: 5px; padding-right: 5px; padding-bottom: 5px; padding-left: 5px; margin-top: auto; margin-right: auto; margin-bottom: auto; margin-left: auto; height: 0px; width: 90%; max-width: 550px;" width="90%" height="0">
+            <tbody style="box-sizing: border-box;">
+              <tr style="box-sizing: border-box;">
+                <td class="container-cell" style="box-sizing: border-box; vertical-align: top; font-size: medium; padding-bottom: 50px;" valign="top">
+                  <table class="c1766" style="box-sizing: border-box; margin-top: 0px; margin-right: auto; background:#728089;margin-left: 0px; padding-top: 5px; padding-right: 5px; padding-bottom: 5px; padding-left: 5px; width: 100%; min-height: 30px; background:#fff; min-height: 150px; margin-bottom: 20px; border-top-left-radius: 3px; border-top-right-radius: 3px; border-bottom-right-radius: 3px; border-bottom-left-radius: 3px;" width="100%" height="0">
+                    <tbody style="box-sizing: border-box;">
+                      <tr style="box-sizing: border-box;">
+                        <td class="cell c1776" style="box-sizing: border-box; width: 70%; vertical-align: middle;" width="70%" valign="middle">
+                          <div class="c1144" style="box-sizing: border-box; padding-top: 10px; padding-right: 10px; padding-bottom: 10px; padding-left: 10px; font-size: 17px; font-weight: 500;text-align: center;color: #fff;">Court
+                            Appointed Special Advocate (CASA) / <%= @casa_organization&.name %>
```

Comment:
It looks like you've defined `@casa_org` in the mailer, but you are using `@casa_organization` here. Since the organization should always exist, you don't need `&.` and can just do `@casa_organization.name`.

Reply:
Yeah I used `@casa_org` so I changed that to match in the view here so it renders correctly.

Reply:
I went ahead and changed this in both spots to reflect what's in my mailer. (: